packages/terra-overlay/src/OverlayContainer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+import './Overlay.scss';
+
+const propTypes = {
+ /**
+ * The overlay and the elements the overlay should hide when overlay isRelativeToContainer and isOpen.
+ */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const OverlayContainer = ({ children, ...customProps }) => {
+  const attributes = Object.assign({}, customProps);
```

Comment:
Since you're not manipulating the customProps object here, you don't need to clone it.